### PR TITLE
improve caching code sample

### PR DESCRIPTION
**Pull request by herzrasen**
This PR is meant to improve the code snippets provided for the caching module as mentioned in #2060

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
CI failure is just the code formatting check, if you make sure to compile before checking in the code will be automatically formatted for you, so that needs a separate commit now.

**Comment by herzrasen** [comment]
@johanandren ok sorry, forgot that. recompiled the project and committed the updated sources.

File: docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 package docs.http.scaladsl.server.directives
 
+import akka.http.scaladsl.model.HttpMethods
```
Comment (jlprat) [inline docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala]: Usually is better to have the imports within the examples, so they can be copy-pasted and run without needing to guess the right import.

Comment (herzrasen) [reply docs/src/test/scala/docs/http/scaladsl/server/directives/CachingDirectivesExamplesSpec.scala]: @jlprat you are absolutely right. In this case the import is unused. The first implementation checked if the Request was an HTTP GET. Forgot the remove it.

**Pull request merged**